wlauto/workloads/googlephotos/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -413,4 +416,17 @@ private void saveAndReturn() throws Exception {
         navigateUpButton.waitForExists(viewTimeout);
         navigateUpButton.click();
     }
+
+    // Helper to tag an individual photograph based on index in wa-working gallery.
```

Comment:
Maybe I can change this to 'mark' or 'check'. After long clicking it should mark the photo with a little tick in the corner of the image to demonstrate it has been selected. I can add a fuller comment also.